docs/data.rst

```diff
@@ -1099,6 +1099,43 @@ Response
     {"status_code": 200, "message": "3 records were deleted"}
 
 
+Permanent Deletion of Submissions
+------------------------------------
+
+**Permanently Delete a specific submission instance**
+
+`DELETE /api/v1/data/{pk}/{dataid}`
+
+A POST payload of parameter `permanent_delete` with the value 'True'. The value is 'False' by default.
+
+**Payload**
+::
+
+    permanent_delete = 'True'
+
+
+Response
+^^^^^^^^^
+
+::
+    HTTP 204 No Content
+
+**Permanently Delete a subset of submissions**
+
+`DELETE /api/v1/data/{pk}?permanent_delete=true&instance_ids=1,2,3`
```

Comment:
We also support passing this values as payloads via `curl -X DELETE https://api.ona.io/api/v1/data/28058' -d 'instance_ids=101425,108428,1974624' -d permanent_delete='True'`. Do we need the query params?

Reply:
The `permanent_delete='True'` should allow permanent deletion of submissions when deleting a single or a subset of submissions